plugins/main/bridge/bridge.go

```diff
@@ -53,6 +57,8 @@ type NetConf struct {
 	HairpinMode  bool   `json:"hairpinMode"`
 	PromiscMode  bool   `json:"promiscMode"`
 	Vlan         int    `json:"vlan"`
+	Prune        bool   `json:"prune"`
+	LockDir      string `json:"lockDir"`
```

Comment:
What's this field used for? Seems useless...

Reply:
optional parameter to specify directory where plugin configure lock file.
We could remove it and change always use default dir. Any preferences?

Reply:
Oh... a mistake, maybe we should put this in docs..